x-pack/plugins/observability/server/utils/queries.ts

```diff
@@ -19,8 +19,8 @@ export function alertStatusQuery(status: AlertStatus) {
 }
 
 export function rangeQuery(
-  start?: number,
-  end?: number,
+  start?: number | string,
+  end?: number | string,
```

Comment:
Hmm.. why should this accept strings?

Reply:
This es query is also valid with a string. Does APM have a preference to only query date range with numbers?

Reply:
It does seem like this is adding unnecessary complexity. We have up until now always used numbers for the timestamp. Do we need to change this now or can we avoid it?

If the query we're doing does not follow the semantics of every other use of `rangeQuery` maybe it would be better to not use this function at all.